.gitignore

```diff
@@ -1,3 +1,4 @@
+.idea
```

Comment:
this change should be reverted (you should add it to your local git ignore file instead)